ports/raspberrypi/Makefile

```diff
@@ -235,7 +242,7 @@ SRC_SDK := \
 	$(SRC_SDK_CYW43) \
 
 SRC_SDK := $(addprefix sdk/, $(SRC_SDK))
-$(patsubst %.c,$(BUILD)/%.o,$(SRC_SDK) $(SRC_CYW43)): CFLAGS += -Wno-missing-prototypes -Wno-undef -Wno-unused-function -Wno-nested-externs -Wno-strict-prototypes -Wno-double-promotion -Wno-sign-compare -Wno-unused-variable -Wno-strict-overflow -Ilib/cyw43-driver
+$(patsubst %.c,$(BUILD)/%.o,$(SRC_SDK) $(SRC_CYW43) $(SRC_TCA9555R)): CFLAGS += -Wno-missing-prototypes -Wno-undef -Wno-unused-function -Wno-nested-externs -Wno-strict-prototypes -Wno-double-promotion -Wno-sign-compare -Wno-unused-variable -Wno-strict-overflow -Ilib/cyw43-driver
```

Comment:
We usually prefer only to silence warnings in code that comes from submodules (like files from pico-sdk). In files that we add to the core, the warnings should be addressed, so that it's not necessary to list the file here.

Reply:
Ah thanks. I admit it's inclusion here was purely because the SRC_CYW43 was included there, without really understanding what the line was doing. I will remove.